components/brave_wallet/browser/solana_transaction_unittest.cc

```diff
@@ -323,6 +351,13 @@ TEST_F(SolanaTransactionUnitTest, FromSignedTransactionBytes) {
   EXPECT_NE(*result, transaction);
   EXPECT_EQ(*result->message(), *transaction.message());
   EXPECT_EQ(result->raw_signatures(), signatures);
+
+  // Signed transaction bytes with empty signatures
+  result = SolanaTransaction::FromSignedTransactionBytes(
+      valid_signed_tx_with_two_empty_signatures);
```

Comment:
Aside from this test case, I'm curious of how real tx returned by Jupiter usually look like in terms of signatures, could you share so we can discuss if there are other cases we want to make sure we have the coverage? Thanks.

Reply:
As discussed in DM, I changed the test to use a transaction with single signature in order to better reflect the reality with Jupiter.

a6612754dd3093e1d1d5b608f74c012cb74ba56a